pulsar-client-admin-api/src/main/java/org/apache/pulsar/client/admin/Topics.java

```diff
@@ -361,6 +361,42 @@
      */
     CompletableFuture<Void> createPartitionedTopicAsync(String topic, int numPartitions);
 
+    /**
+     * Create a partitioned topic.
+     * <p/>
+     * Create a partitioned topic. It needs to be called before creating a producer for a partitioned topic.
+     * <p/>
+     *
+     * @param topic
+     *            Topic name
+     * @param numPartitions
+     *            Number of partitions to create of the topic
+     * @param createLocalTopicOnly
+     *            False indicate create topic in all replicate clusters,
+     *            true indicate only create topic in local cluster.
+     * @throws PulsarAdminException
+     */
+    void createPartitionedTopic(
```

Comment:
this is a new public API for users.

I am not sure we should add this here.

it is fine to have it in the Implementation but we should not give this to the users

@merlimat WDYT ?

Reply:
Yes, I think this should be a broker internal behavior, the param `createLocalTopicOnly` shouldn't be specified by users. @codelipenghui Please confirm.

Reply:
Yes, we don't need to expose to users, keep it in the implementation is a better solution